src/tcp-redirections/cc-tcp-redirection-form.smart.js

```diff
@@ -78,25 +60,25 @@ defineComponent({
         component.redirections = null;
 
```

Comment:
At some point, we need to reset the `error` property too
`component.error = null;`

Should we do that here or at line 39?

Reply:
I don't understand this remark for this example as like 59 already has a reset for the error property.

Reply:
Sorry for this one